iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -330,6 +333,16 @@ public boolean canProvideBasicStatistics() {
     return true;
   }
 
+  @Override
+  public StorageFormatDescriptor getLoadDataStorageFormatDescriptor(org.apache.hadoop.hive.metastore.api.Table tbl)
```

Comment:
could we simplify the name to `getStorageFormatDescriptor`?

Reply:
done